docs/api-transforms.rst

```diff
@@ -95,6 +95,12 @@ Spectrogram
 .. autoclass:: Spectrogram
     :members:
 
+Log
+---
+
+.. autoclass:: Log
+    :members:
+
```

Comment:
It looks to me that `Standardize` is missing here and `standardize` in the functional part of the documentation.

Reply:
Yes, I uploaded it in the latest commits.